packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -84,19 +84,21 @@ class MethodChannelCamera extends CameraPlatform {
   }
 
   @override
-  Future<int> createCamera(
+  Future<int> createCameraWithSettings(
```

Comment:
I think removing `createCamera` is a breaking change for unendorsed packages. You should keep `createCamera` and have it call `createCameraWithSettings` with default settings.

Reply:
Restored `createCamera`. Called `createCameraWithSettings`.